sketcherMinimizer.h

```diff
@@ -109,6 +109,7 @@ class EXPORT_COORDGEN sketcherMinimizer
     static void canonicalOrdering(sketcherMinimizerMolecule* minMol);
 
     // void initializeFromMolecule(ChmMol& mol);
+    void writeMinimizationData(sketcherMinimizerPointF center, float s, float c);
```

Comment:
What do `s` and `c` mean?

Reply:
Good question, I think they are the x and y values that define the rotation of the coordinates in 2D space. They are calculated on line 993 of `sketcherMinimizer.cpp`.

When I first tried writing coordinates to a file, I did not apply this rotation and was very confused why the coordinates I printed to a file were rotated versions of the actual coordinates -- turns out we have to perform the rotation :)

Reply:
sine and cosine of the rotation angle :)